combat-trainer.lic

```diff
@@ -2259,16 +2261,33 @@ class AttackProcess
       game_state.loaded = false
       waitrt?
     elsif game_state.loaded
-      actions = game_state.next_aim_action.split(';')
-      actions.each do |action|
-        first_word = action.split(' ')[0].downcase
-        matches = [/^You /]
-        matches = ['You get', 'You draw out', 'You pick up', "You're already holding that"] if @get_actions.include?(first_word)
-        matches = ['Roundtime',"You aren't close enough to attack"] if @rt_actions.include?(first_word)
-        matches = ['You put', 'You sheathe'] if @stow_actions.include?(first_word)
-        bput(action.strip, matches)
-        pause 0.25
-        waitrt?
+      if game_state.use_offhand_trainables?
+        skill = game_state.determine_offhand_weapon
+        skill = 'Brawling' if game_state.current_weapon_skill.eql?'Bow'
+        if skill.eql?'Brawling'
+          3.times{bput('gouge left', 'You')}
+          return
+        end
+
+        weap = game_state.wield_offhand_weapon(skill)
+        %w[lob jab].each do |action|
+          result = bput("#{action} left", 'Roundtime', 'close enough')
+          fput('eng') if result =~ /close enough/i
+          bput("get my #{weap}", 'You pick up') if action.eql?'lob'
```

Comment:
I _think_ I may have had an issue at some point when trying to pick up a bonded weapon instead of using `invoke`

Reply:
hm, you can pick up or invoke any weapon, this seems terribly weird if it's a bug. :( I don't know how to handle that with the system's current implementation. I'd need to add in a method to handle getting a weapons current info from game_states equipment manager.